Data/crypto/readme.md

```diff
@@ -0,0 +1,127 @@
+![alt tag](https://raw.githubusercontent.com/QuantConnect/Lean/master/Documentation/logo.white.small.png) 
+## LEAN Data Formats / Cryptocurrency (crypto)
+
+QuantConnect hosts crypto data provided by [Kaiko](https://www.kaiko.com).
+The data contains both *Trade* and *Quote* data. Using the ToolBox applications `GDAXDownloader` and `BitfinexDownloader`, you can obtain historical *trade* data for free, but not *quote* data with this method. 
+You can also download crypto data (trades and quotes starting with Tick and ending with Minute resolution) for a fee on our website. You can explore the data and purchase it at https://www.quantconnect.com/data/tree/crypto
+
+CSV files are stored in compressed zip files, each containing a single CSV file.
+
+Crypto data supports the following Resolutions:
+
+* Tick
+* Second
+* Minute
+* Hour
+* Daily
+
+The markets we currently support are: 
+
+* GDAX/Coinbase Pro - 24/7 Market
+* Bitfinex (Beta) - 24/7 Market
+
+For crypto data, `tickType` can be:
+
+* trade
```

Comment:
Aim of these readme's is to describe the *format* of the files, not the features of LEAN. What do people need to know to **write** crypto data?

Reply:
Agreed, I thought it was necessary to provide some context for what `tickType` means, since I use it as a placeholder in the filename format.